packages/components/psammead-image-placeholder/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Psammead Image Placeholder Changelog
 
+<!-- prettier-ignore -->
```

Comment:
Instead of having this here, can you put an `excludeFiles` matching all `CHANGELOG.md` files in the global prettier config

Reply:
Could do, but it's kinda useful having Prettier run across the rest of the file. It's only the table that it borks things for. I'd sooner have this comment as part of the `CHANGELOG.md` template 🤔

Reply:
But this currently only runs prettier on the first 2 lines of the file.. :/

Reply:
Does it? My understanding was that it only ignores the next 'block':

> A JavaScript comment of // prettier-ignore will exclude the next node in the abstract syntax tree from formatting.

https://prettier.io/docs/en/ignore.html#javascript